src/modules/misc.ts

```diff
@@ -218,6 +225,28 @@ export async function handleJustAskCommand(
 	}
 }
 
+export async function handleJustAskSlashCommand(
+	interaction: Discord.ChatInputCommandInteraction
+): Promise<void> {
+	if (interaction.channel) {
+		await sendJustAsk(
+			interaction,
+			interaction.channel.send.bind(interaction.channel)
```

Comment:
Why not this?
```suggestion
			message => interaction.channel.send(message)
```

While a little more verbose, it's way easier to read